_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,231 @@
+---
+id: a25f45
+name: headers attribute only refers to cells in the same table element.
+rule_type: atomic
+description: |
+  This rule checks that the headers attribute must only refer to cells in the same table element.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to `td` elements with a `headers` attribute, that are [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Expectation
+
+Each target element refers in its `headers` attribute to other `th` elements or cells with a [semantic role](#semantic-role) of `columnheader` or `rowheader` within the same `table` element.
+
+## Assumptions
+
+- This test assumes that the `headers` attribute is only used to identify table headers. If other information is included in the `headers` attribute, the rule may fail on issues that are not accessibility concerns. For example, if `headers` is used to include information for script, this rule may not be accurate.
+- Tables using `headers` id associations are intended as data rather than [layout table](#layout-table).
```

Comment:
I don't think you need to define layout table here. Linking out to something about layout tables in WCAG is enough. We may need that term for some other rule.

Reply:
Agreed with @WilcoFiers, to leave this as is for now.